osidb/models.py

```diff
@@ -444,7 +445,16 @@ def fts_search(q):
         # If search has no results, this will now return an empty queryset
 
 
-class Flaw(WorkflowModel, TrackingMixin, NullStrFieldsMixin, AlertMixin, ACLMixin):
+class Flaw(
+    ACLMixin,
+    AlertMixin,
+    # Bugzilla sync must take place after
+    # the validations not to bypass them
```

Comment:
This makes me a bit uneasy, maybe `BugzillaSync` shouldn't be a mixin but a subclass, WDYT? You can have alerting/validation without bzsync but you cannot have bzsync without alerting/validation

Reply:
@Elkasitu actually I am not sure that my comment is actually up-to-date here. You can see in `BugzillaSyncMixin` that the first thing it does actually is the call of `super().save(*args, **kwargs)` which means that no matter how the mixins are ordered the validations will run. I will update the comments and to be really safe add a test case specifically checking that the validations are run when syncing to Bugzilla.

Reply:
Done.